spec/presenters/admin_artist_list_spec.rb

```diff
@@ -79,4 +71,62 @@
       expect(artist_list.pluck(:updated_at)).to be_monotonically_decreasing
     end
   end
+
+  describe '.csv' do
+    let(:parsed) { CSV.parse(list.csv, headers: true) }
+
+    its(:csv_filename) { is_expected.to eql 'mau_artists.csv' }
+
+    it 'has some data in the csv' do
+      expect(parsed.first['Full Name']).to eql list.send(:artists).first.full_name
+      expect(parsed.first.key?('Shop Url')).to be true
+    end
+
+    describe 'when there is a current open studios with participants' do
```

Comment:
these tests are pretty gross but I don't know a better way to get it tested.  Setup is complicated and verification is complicated - especially because we have timestamps in there.